src/common/popover/PopoverOverlay.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Popover from 'react-bootstrap/lib/Popover';
+import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
+import PropTypes from 'prop-types';
+
+const PopoverOverlay = ({
+  content, title, trigger, children, placement
+}) => (
+  <div className="app-PopoverOverlay">
+
```

Comment:
Why does this component have so many empty lines in it?

Reply:
Fixed